input/fsh/VS_Staging.fsh

```diff
@@ -146,42 +134,23 @@ Description: "Staging system or method used for staging cancers. The terms in th
 * SCT#254383006 "International Federation of Gynecology and Obstetrics staging system of gynecological malignancy (tumor staging)"
 * SCT#254385004 "International Federation of Gynecology and Obstetrics uterine tumor staging system (tumor staging)"
 * SCT#254384000 "International Federation of Gynecology and Obstetrics vaginal tumor staging system (tumor staging)"
-* SCT#254363005 "International neuroblastoma staging system (tumor staging)"
 * SCT#1149162008 "International Staging System for multiple myeloma (staging scale)"
 * SCT#254380009 "Marshall Modification of Jewett-Strong staging system for urinary bladder tumor (tumor staging)"
 * SCT#254362000 "Modified Astler-Coller staging system (tumor staging)"
 * SCT#385359000 "Modified Dukes staging system (tumor staging)"
-* SCT#254364004 "National Wilms' tumor study staging system (tumor staging)"
 * SCT#1149131009 "Rai staging system for chronic lymphocytic leukemia (tumor staging)"
 * SCT#1149163003 "Revised International Staging System for multiple myeloma (staging scale)"
-* SCT#254365003 "Siopel liver staging system (tumor staging)"
-* SCT#254370005 "St Jude's malignant histiocytosis tumor staging system (tumor staging)"
-* SCT#254371009 "St Jude's non-Hodgkin's lymphoma staging system (tumor staging)"
 * SCT#254366002 "Stannard staging system for retinoblastoma (tumor staging)"
 * SCT#254382001 "Trophoblastic malignancy staging system (tumor staging)"
-* SCT#254368001 "United Kingdom children's cancer study group central nervous system tumor staging system (tumor staging)"
 * SCT#254378003 "Walter Reed testicular tumor staging system (tumor staging)"
 * SCT_TBD#USCRS-34949 "Enneking surgical staging system (tumor staging)" // requested June 2023
```

Comment:
I thought we were dropping this

Reply:
For the record, this VS is like a superset of what we might do at some later date. Including or excluding any codes reallly has no effect on the use or functionality of mCODE.

Reply:
Keep the requests going. We might need all of them, including the pediatric and surgical staging systems.